src/sass/_appbar.scss

```diff
@@ -0,0 +1,41 @@
+.appbar{
```

Comment:
if you're worried about how bootstrap is global, we should avoid doing it here too.  If you nest everything under a `.wood` class, then you can allow people to be more target width their integration:

    .wood {
        .appbar {
            ...
         }
    }

this way, if you're using wood for your whole site, you can just do `<body class="wood">` but if not, you can be more specific.